tinytga/src/lib.rs

```diff
@@ -1,4 +1,82 @@
-//! No-std compatible TGA parser designed for embedded systems, but usable anywhere
+//! A small TGA parser designed for embedded, no-std environments but usable anywhere. Beyond
+//! parsing the image header, no other allocations are made.
+//!
+//! To access the individual pixels in an image, the [`Tga`] struct implements `IntoIterator`. It is
+//! also possible to access the raw image data by reading the [`pixel_data`] field.
```

Comment:
I think this doesn't make it clear enough that `pixel_data` might return data that is still RLE encoded.

Reply:
I'm not sure how to make "raw" clearer. "raw image **file** data"? "raw, **possibly still compressed** image data"?